src/main/kotlin/org/veupathdb/service/eda/compute/plugins/PluginProvider.kt

```diff
@@ -43,4 +44,7 @@ interface PluginProvider<R : ComputeRequestBase, C> : PluginMeta<R> {
    */
   fun getContextBuilder(): PluginContextBuilder<R, C> =
     PluginContextBuilder()
+
+  fun getConstraintSpec(): ConstraintSpec = ConstraintSpec()
```

Comment:
Maybe a simple comment here like "Override this method to add variable and collection constraints to your plugin's input"

Reply:
That's correct, right?